src/snowflake/connector/network.py

```diff
@@ -907,14 +920,14 @@ def _request_exec_wrapper(
             return {}
 
     def log_and_handle_http_error_with_cause(
-            self,
-            e: Exception,
-            full_url: str,
-            method: str,
-            retry_timeout: int,
-            retry_count: int,
-            conn: SnowflakeConnection,
-            timed_out: bool = True
+        self,
+        e: Exception,
+        full_url: str,
+        method: str,
+        retry_timeout: int,
+        retry_count: int,
+        conn: SnowflakeConnection,
+        timed_out: bool = True,
```

Comment:
Do we need a merge gate checking for format? It is a bit weird that this PR contains unrelated format changes.